integration_tests/tests/apply/risks_and_needs/risk-to-self/current_risk.cy.ts

```diff
@@ -58,7 +57,9 @@ context('Visit "Risks and needs" section', () => {
   //  Scenario: view current risk questions
   //    Then I see the "current risk" page
   it('presents current risk page', function test() {
-    Page.verifyOnPage(CurrentRiskPage, this.application)
+    const page = Page.verifyOnPage(CurrentRiskPage, this.application)
+
+    page.shouldShowOasysImportDate(this.application.data['risk-to-self']['current-risk'].dateOfOasysImport)
```

Comment:
Just a small point, but I wonder if it would be easier on the eye and mind to either:

1. delegate all the implementation detail to the test helper e.g. `page.shouldShowOasysImportDate('risk-to-self', 'current-risk')`, or
2. use a literal value (having set it in the test, presumably to a unique date?) e.g. `page.shouldShowOasysImportDate('2023-08-29')`

Reply:
agree that's an improvement, thanks!